Nitaicandra/CENPROJECT
# Business profile + Review Feature

## Summary of changes
- business profile
- ability to read, write, and reply to reviews
- added book and view business profile functionality for search results

## How to test
Customer-side
1. Log in as a customer and create a booking in the past (you might need to do this via postman)
2. Go to /home and you should see it listed there with a "review" button. Click
3. Fill in the form and click save. If successful, you will be redirected back to /home.
4. Now visit `http://localhost:5173/business/businessId` (replace businessId by the actual id of the business you booked). This is the business profile page.
5. Try to click on the business email. A window should open with your email client installed on your computer.
6. Scroll down the page and check if you see your new review listed.
7. Click on "view details" for one of the services listed in the page. Try the buttons that should now be functional.

Business-side
1. Log in as a business and go to its profile page using `http://localhost:5173/business/businessId`
2. scroll down to the reviews. You should be able to reply to the review you left previously
3. Upon posting, the page should reload and you will see your reply. You wont be able to reply to it again.
